src/bidiMapper/domains/context/PreloadScriptStorage.spec.ts

```diff
@@ -199,33 +99,23 @@ describe('PreloadScriptStorage', () => {
         filterDescription: 'bidi id',
         filter: {id: MOCKED_UUID_1},
       },
+      {
+        filterDescription: 'target id',
+        filter: {targetId: CDP_TARGET_ID},
+      },
     ].forEach(({filterDescription, filter}) => {
       it(`find preload scripts in ${contextDescription} by ${filterDescription}`, () => {
-        preloadScriptStorage.addPreloadScripts(
-          null,
-          [cdpPreloadScript1],
-          functionDeclaration,
-          sandbox
-        );
+        const preloadScript1 = createPreloadScript(MOCKED_UUID_1, null);
+        preloadScriptStorage.addPreloadScript(preloadScript1);
 
-        expect(preloadScriptStorage.findPreloadScripts(filter)).to.deep.equal([
-          {
-            id: MOCKED_UUID_1,
-            cdpPreloadScripts: [cdpPreloadScript1],
-            contextId: null,
-            functionDeclaration,
-            sandbox,
-          },
-        ]);
+        expect(preloadScriptStorage.findPreloadScripts(filter)[0]).to.equal(
```

Comment:
Could you keep the same pattern as it used to be previously, by comparing the whole array, instead of only `[0]`?

Reply:
done